### DISCUSSION TITLE
Can tgcf handle break in chat id in source

### DISCUSSION THREAD

**Discussion by kaushalruparel** [aahnik/tgcf#308] [Q&A] [answered]
There can be a break in chat ids in channels if some message is deleted. For eg. if we create a new channel and broadcast 3 messages and then delete the 2nd message, there will be a break in consecutive chat id. Does tgcf handle this case or it errors out when we use past mode?

**Comment by tissole** [comment, marked as answer]
The script will work in that case without any problems. The msgs could be 1,3,4,8,9 etc or 5,6,7,10,35 etc, the script will forward them with no errors.

**Reply by kaushalruparel** [reply]
Thank you!
